lib/babbler.rb

```diff
@@ -12,13 +12,11 @@ def babble(seed = nil)
 
       babble_words = []
 
-      Babbler.config.num_adjectives.times do
-        babble_words.push(adjectives[prng.rand(adjectives.length)])
+      Babbler.config.format.gsub(' ', '').split('').each do |f|
+        babble_words.push(send("format_#{f}")[prng.rand(adjectives.length)])
```

Comment:
the above also uses collect instead of creating array and pushing onto it; also means you don't need strip at bottom.